docs/source/_json/history_get.resp

```diff
@@ -0,0 +1,36 @@
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+[
+  {
+    "action": null, 
```

Comment:
@jaroel I guess the null value for content creations comes from CMFEditions. This should be "Created" in my opinion. plone.restapi might be the wrong level to fix this though.